### DISCUSSION TITLE
Config

### DISCUSSION THREAD

**Discussion by Saviox0** [jagrosh/MusicBot#651] [Questions] [answered]
I want to ask is there any way to make the bot only listen to my commands

**Comment by DemonFiend** [comment, marked as answer]
could just make yourself the only one able to chat in a specific channel that the bot is using.

**Reply by jagrosh** [reply]
This is the correct way to do this; more specifically, make a private channel and use the `settc` command with that channel so that the bot will only allow music commands in that private channel.
